src/core/room-lifecycle-manager.ts

```diff
@@ -11,10 +11,9 @@ import { DefaultRoomLifecycle, NewRoomStatus, RoomStatus, RoomStatusChange } fro
  */
 export interface ContributesToRoomLifecycle extends HandlesDiscontinuity {
```

Comment:
This will be useful for logging and reporting errors inside roomLifeCycleManager. 
`featureName()` will return values `messsages`, `presence`, `typing`, `occupancy`, `reactions` etc, wdyt

Reply:
Since, I am implementing `src/core/room-lifecycle-manager.ts` as is in `chat-kotlin`, I found this to be useful